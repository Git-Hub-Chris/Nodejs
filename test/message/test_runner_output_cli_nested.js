// Flags: --no-warnings --test test/fixtures/test-runner/nested.js
// FlagsOnly
'use strict';
require('../common');
