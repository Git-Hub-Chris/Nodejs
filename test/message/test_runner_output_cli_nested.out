TAP version 13
# Subtest: *
    # Subtest: level 0a
        # Subtest: level 1a
        ok 1 - level 1a
          ---
          duration_ms: *
          ...
        # Subtest: level 1b
        not ok 2 - level 1b
          ---
          duration_ms: *
          failureType: 'testCodeFailure'
          error: 'level 1b error'
          code: 'ERR_TEST_FAILURE'
          stack: |-
            *
            *
            *
            *
            *
            *
            *
            *
            *
            *
          ...
        # Subtest: level 1c
        ok 3 - level 1c # SKIP aaa
          ---
          duration_ms: *
          ...
        # Subtest: level 1d
        ok 4 - level 1d
          ---
          duration_ms: *
          ...
        1..4
    not ok 1 - level 0a
      ---
      duration_ms: *
      failureType: 'subtestsFailed'
      error: '1 subtest failed'
      code: 'ERR_TEST_FAILURE'
      ...
    # Subtest: level 0b
    not ok 2 - level 0b
      ---
      duration_ms: *
      failureType: 'testCodeFailure'
      error: 'level 0b error'
      code: 'ERR_TEST_FAILURE'
      stack: |-
        *
        *
        *
        *
        *
        *
        *
      ...
    1..2
not ok 1 - *
  ---
  duration_ms: *
  failureType: 'subtestsFailed'
  exitCode: 1
  error: '0 subtests failed'
  code: 'ERR_TEST_FAILURE'
  ...
1..1
# tests 1
# pass 0
# fail 1
# cancelled 0
# skipped 0
# todo 0
# duration_ms *
